src/backend/aspen/api/utils/lineage.py

```diff
@@ -0,0 +1,42 @@
+import re
+from typing import List
+
+import sqlalchemy as sa
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from aspen.database.models import PangoLineage
+
+"""
+Takes in a list of lineages and wildcard lineage expressions (as strings) and
+returns a list of specific lineages. Wildcards are expanded to include both
+the base expression and all sublineages.
+
+Example:
+We encounter the wildcard expression 'BA.1.*'.
```

Comment:
Very minor: I was a bit on the fence whether ppl would interpret `BA.1.*` to include `BA.1` or `BA.1*` to include `BA.1`, and noticed here: https://github.com/cov-lineages/pango-designation/blob/master/pango_designation/alias_key.json they use `BA.1*`. I think non-coding ppl may be less likely to expect `BA.1*` to include `BA.11`?

Reply:
That makes sense, I'll update it to keep it consistent with what's in the `cov-lineages` repo. My coder brain definitely interprets `BA.1*` to include `BA.11`